attributes/default.rb

```diff
@@ -21,7 +21,7 @@
 default['rocketchat']['group'] = 'rocketchat'
 default['rocketchat']['install_dir'] = '/srv/rocketchat'
 
-default['rocketchat']['url'] = 'https://rocket.chat/releases/latest/download'
```

Comment:
> [edit] the only reason to maybe hold back to a specific release would be the checksum line below maybe...

Bingo.  We were trying to get RocketChat up and installed.  Based on the checksum, the assumption was config templates were correct; not knowing if there were config changes between the different 0.56.0 and latest versions this seemed the safest choice.  (also, I think there was a comment about updating to v0.56.0)

[I had started on a refactor](https://github.com/maraaaa/chef-rocketchat/tree/maraaaa/v0.2.2-support-ubuntu-1604) but ran [into some issues](https://travis-ci.org/maraaaa/chef-rocketchat/jobs/236841602) with building the rocket package with non-binary versions of node, (and then real life got in the way...).